pkg/network/node/node.go

```diff
@@ -300,6 +300,7 @@ func (node *OsdnNode) Start() error {
 	if err != nil {
 		return fmt.Errorf("failed to get network information: %v", err)
 	}
+	node.oc.vxlanPort = node.networkInfo.VXLANPort
```

Comment:
Ick. It's bad style to set another object's fields for it. You should pass the port to `SetupSDN`, which can pass it to `oc.SetupOVS`.

Wait, you're already passing it to `oc.SetupOVS`... Oh, I guess the problem is that `oc.AlreadySetUp` needs it before `SetupOVS` has been called... I guess you should just pass the port to `AlreadySetUp` too then...